data/json/items/tool/science.json

```diff
@@ -0,0 +1,129 @@
+[
+  {
+    "id": "chemistry_set",
+    "type": "TOOL",
+    "name": "chemistry set",
+    "description": "This is a chemistry set stored in a box.  The contents include glass containers, hoses, metal wire, a hotplate, and safety glasses.  It might be used to craft some chemistry projects if you're so inclined.",
+    "weight": 5200,
+    "volume": 18,
+    "price": 20000,
+    "to_hit": -5,
+    "material": [ "glass", "cotton" ],
+    "symbol": ";",
+    "color": "light_gray",
+    "ammo": "battery",
+    "sub": "hotplate",
+    "max_charges": 200,
+    "charges_per_use": 1,
+    "qualities": [ [ "DISTILL", 1 ], [ "CHEM", 3 ], [ "BOIL", 1 ] ],
+    "use_action": "HOTPLATE"
+  },
+  {
+    "id": "chemistry_set_basic",
+    "type": "TOOL",
+    "category": "tools",
+    "name": "basic chemistry set",
+    "description": "This is a basic chemistry set which includes glass containers, hoses and safety glasses.  It might be used to craft some chemistry projects if you're so inclined, but you'll need a source of heat.",
```

Comment:
The metal wire in the original chemistry set was to keep the glass containers in a place. Either mention it in both descriptions or not mention it at all, as it serves only that purpose ;)